chia/simulator/setup_services.py

```diff
@@ -71,7 +71,7 @@ def get_capabilities(disable_capabilities_values: Optional[List[Capability]]) ->
                 disable_capabilities_values.remove(Capability.BASE)
 
             updated_capabilities = []
-            for capability in capabilities:
+            for capability in default_capabilities[NodeType.FULL_NODE]:
```

Comment:
this looks suspicious. Isn't this supposed to be the list of all known capabilities?
`get_capabilities()` probably needs to take the node type

Reply:
yes, this pr does not add the passing of capabilities to services  other then the full node, i think maybe we should add that code even if its not used yet, if i understand we want to enable other implementations to use  capabilities, they should be able to do that without touching "setup_services"

Reply:
setup_services is only for the simulator from what I can see - start_service does use the node type to determine which capabilities

Reply:
Just added the `node_type` to `get_capabilities`. Everywhere else should be using `default_capabilities[type]` already, including the tests